recipes-app/iot2050-firmware-update/files/iot2050-firmware-update.tmpl

```diff
@@ -702,17 +758,20 @@ def main(argv):
         The Device may become brick.
         Please try to upgrade firmware again!
         ====================================
-Hit any Key to Exit:
+Hit the Enter Key to Exit:
         '''
         # Rollback the firmware when the upgrade fails
         try:
-            print('Warning: Upgrade failed, rollback started')
-            updater.rollback_firmware()
+            print(BOLD + 'Warning: Upgrade failed, rollback started' + ENDC + '\n')
+            ret = updater.rollback_firmware()
+            if ErrorCode.ROLLBACK_FAILED.value == ret:
+                return ret
         except UpgradeError:
             print("Rollback failed")
 
-        input(intput_reminder)
-
+        the_input(False, intput_reminder)
```

Comment:
Why do not we use args.quiet instead of False? Also, what is INTERNAL_ERR? Is it an error during flashing process? I think we should use a more descriptive name.

Reply:
If we provide False, then the code will still wait for an input in quiet mode too.